cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/StorageClient.java

```diff
@@ -47,4 +47,12 @@ public interface StorageClient<T> {
    * @return the root prefix for OpenHouse on the storage system
    */
   String getRootPrefix();
+
+  /**
+   * Checks if the path exists on the storage system
+   *
+   * @param path
+   * @return true if path exists else false
+   */
+  boolean pathExists(String path);
```

Comment:
Can you please add comment about the expected format of path? Should it include scheme or not? Is it URI-style path? Or is it absolute path starting from root folder or bucket?

Reply:
done